src/time-resolution.jl

```diff
@@ -205,10 +210,7 @@ compute_rp_partition([partition1, partition2], :highest)
  11:12
 ```
 """
-function compute_rp_partition(
-    partitions::AbstractVector{<:AbstractVector{<:UnitRange{<:Integer}}},
-    strategy,
-)
+function compute_rp_partition(partitions, strategy)
```

Comment:
I removed the type because it gave an error before, but maybe now it works with the type?